gui/middleware/notifier.py

```diff
@@ -448,6 +446,9 @@ def geli_passphrase(self, volume, passphrase, rmrecovery=False):
         Raises:
             MiddlewareError
         """
+        from freenasUI.storage.models import Volume
```

Comment:
Why the relative import here?

Reply:
You mean lazy import? Because of recursive imports on notifier.